app/views/map/_peopleLeaflet.html.erb

```diff
@@ -12,17 +12,36 @@
 	<% end %>
 
   <script>
-	var map<%= unique_id %> = L.map('map<%= unique_id %>').on('load', onMapLoad).setView([<%= lat %>,<%= lon %>], 2);
- 	L.tileLayer("//a.tiles.mapbox.com/v3/jywarren.map-lmrwb2em/{z}/{x}/{y}.png").addTo(map<%= unique_id %>);
- 	function onMapLoad(e){
-     	$.getJSON("/api/srch/peoplelocations?srchString=100" , function(data){
- 			if (!!data.items){
- 				for (i = 0; i < data.items.length ; i++) { 
- 					var url = data.items[i].docUrl ;
- 					var title = data.items[i].docTitle ;
-   					L.marker([data.items[i].latitude , data.items[i].longitude]).addTo(map<%= unique_id %>).bindPopup("<a href=" +  url + ">" + title + "</a>") ;
-				}
- 			}
- 		});
-	}
+   var map<%= unique_id %> = L.map('map<%= unique_id %>').on('load', onMapLoad).setView([<%= lat %>,<%= lon %>], 2);
+   L.tileLayer("//a.tiles.mapbox.com/v3/jywarren.map-lmrwb2em/{z}/{x}/{y}.png").addTo(map<%= unique_id %>);
+   var searchLayer<%= unique_id %> = L.layerGroup().addTo(map<%= unique_id %>);
```

Comment:
Given `<%=unique_id %>` because we may have more than 1 /people map (inline) on a page . So we need unique variable name 😄 .